File: C#/Estrutura repetitiva/crescente.cs
// Variável
int x, y;

// Entrada de dados
Console.Write("Digite dois números: ");
x = int.Parse(Console.ReadLine());
y = int.Parse(Console.ReadLine());

// Estrutura repetitiva
while (x != y)
{
    // Estrutura condicional
    if (x > y)
    {
        Console.WriteLine("Decrescente, afinal " + x + " é maior que " + y);
    }

    else
    {
        Console.WriteLine("Crescente, afinal " + x + " é menor que " + y);
    }

    // Leitura de dados da estrutura repetitiva
    Console.Write("Digite dois números: ");
    x = int.Parse(Console.ReadLine());
    y = int.Parse(Console.ReadLine());
}

File: C++/Estrutura repetitiva/crescente.cpp
// Biblioteca(s)
#include <iostream>

using namespace std;

int main()
{
    // Variável
    int x, y;

    // Entrada de dados
    cout << "Digite dois numeros: ";
    cin >> x;
    cin >> y;

    // Estrutura repetitiva
    while (x != y) {
        // Estrutura condicional
        if (x > y) {
            cout << "Decrescente, afinal " << x << " e maior que " << y << endl;
        }

        else {
            cout << "Crescente, afinal " << x << " e menor que " << y << endl;
        }

        // Leitura de dados da estrutura repetitiva
        cout << "Digite dois numeros: ";
        cin >> x;
        cin >> y;
    }
    return 0;
}
